workflow/commons/src/main/java/com/datastax/oss/dsbulk/workflow/commons/metrics/MetricsManager.java

```diff
@@ -457,13 +459,31 @@ public <T> Function<Flux<T>, Flux<T>> newTotalItemsMonitor() {
     return upstream -> upstream.doOnNext(item -> totalItems.inc());
   }
 
-  public <T> Function<Flux<T>, Flux<T>> newFailedItemsMonitor() {
+  public Function<Flux<Record>, Flux<Record>> newFailedRecordsMonitor() {
```

Comment:
This change wasn't strictly required but I thought it would make things clear to distinguish the generic "failed items monitor" into 3 monitors: failed records, unmappable statements, and failed read/write results.

Reply:
Given this change, does it make sense to add a paragraph of comment to explain what these three types of "failed items" are?